.github/workflows/deploy-action.yaml

```
@@ -25,8 +25,10 @@ jobs:
     - name: Load Config JS
       env:
         FIREBASE_CONFIG_BASE64: ${{ secrets.TRENDLY_FIREBASE_CONFIG }}
+        ENV_LOCAL_BASE64: ${{ secrets.ENV_LOCAL_FILE }}
       run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)